### PR TITLE
3 4

### PR DESCRIPTION
**Pull request by matt-visentin**

### CODE REVIEW

File: src/components/movie-card/movie-card.jsx [hunk 1]
```diff
@@ -8,4 +10,21 @@ export const MovieCard = ({ movie, onMovieClick }) => {
             {movie.title}
         </div>
     );
+};
+
+MovieCard.PropTypes = {
```
Comment (stratospherique) [inline src/components/movie-card/movie-card.jsx:15]: This line is causing a runtime error in the browser console(check it). It's `ComponentName.propTypes`

**Review by stratospherique — CHANGES_REQUESTED** [review]

**Comment by matt-visentin** [comment]
I see what you mean. I corrected it.

**Review by stratospherique — CHANGES_REQUESTED** [review]

File: src/components/movie-card/movie-card.jsx [hunk 2]
```diff
@@ -8,4 +10,21 @@ export const MovieCard = ({ movie, onMovieClick }) => {
...
+            name: PropTypes.string.isRequired,
+            description: PropTypes.string.isRequired,
+        }),
+        director: PropTypes.shape({
+            name: PropTypes.string.isRequired,
+            bio: PropTypes.string.isRequired,
+            birthyear: PropTypes.number
+        }),
+        description: PropTypes.string.isRequired
+    }).isRequired,
```
[the remark this answers is not in the document]
Comment (matt-visentin) [reply src/components/movie-card/movie-card.jsx]: Sure, thank you. I changed that now.

File: src/components/movie-view/movie-view.jsx [hunk 3]
```diff
@@ -25,4 +25,21 @@ export const MovieView = ({ movie, onBackClick }) => {
...
+            name: PropTypes.string.isRequired,
+            description: PropTypes.string.isRequired,
+        }),
+        director: PropTypes.shape({
+            name: PropTypes.string.isRequired,
+            bio: PropTypes.string.isRequired,
+            birthyear: PropTypes.number
+        }),
+        description: PropTypes.string.isRequired
+    }).isRequired,
```
[the remark this answers is not in the document]
Comment (matt-visentin) [reply src/components/movie-view/movie-view.jsx]: Sure, thank you. I changed that now.

File: src/components/movie-view/movie-view.jsx [hunk 4]
```diff
@@ -1,20 +1,20 @@
+import PropTypes from "prop-types";
+
 export const MovieView = ({ movie, onBackClick }) => {
     return (
         <div>
-            <div>
-                <img src={movie.image} />
-            </div>
+
```
[the remark this answers is not in the document]
Comment (matt-visentin) [reply src/components/movie-view/movie-view.jsx]: The image tag seems not to render the image. I also tried with changing "image" with "imagePath" as I have it in on my db but didn't seem to be effective.

**Review by stratospherique — CHANGES_REQUESTED** [review]

**Pull request merged**